superset-frontend/src/profile/components/UserInfo.tsx

```diff
@@ -19,17 +19,36 @@
 import React from 'react';
 import Gravatar from 'react-gravatar';
 import moment from 'moment';
-import { Panel } from 'react-bootstrap';
-import { t } from '@superset-ui/core';
+import Collapse from 'src/common/components/Collapse';
+import { t, styled } from '@superset-ui/core';
 import { UserWithPermissionsAndRoles } from '../../types/bootstrapTypes';
 
 interface UserInfoProps {
   user: UserWithPermissionsAndRoles;
 }
 
+const StyledContainer = styled.div`
+  .ant-collapse-header {
+    background-color: ${({ theme }) => theme.colors.grayscale.light5};
+    .header {
+      margin-left: ${({ theme }) => theme.gridUnit * -6}px;
+    }
+  }
+  .ant-collapse-content {
+    margin-top: ${({ theme }) => theme.gridUnit * -5}px;
+    border-top: none;
+  }
+  .ant-collapse
```

Comment:
You might not need this CSS... the AntD `collapse` component has a `showArrow` boolean prop.

Looking at the "before" version of the UI, it looks like this <Panel> component has no border, and is not collapsible. We can turn those off with `collapsible` and `bordered` props set to `false`. 

But now that I'm looking at it, I don't think we even need the Collapse component here, do we? It just looks like a white div with rounded corners is all that we need to remove the React Bootstrap component.